### ISSUE TITLE
Connections need configured for RG endpoints when appropriate

### ISSUE THREAD

**Issue by ptpaterson** [rts-rob/fauna-labs#1]
Examples should be clear that if your database is hosted in a Region Group other than "Classic", you need to configure the client connection for your database.  Specifically, the `domain` option needs to be set.

```javascript
const client = new Client({
  secret: YOUR_SECRET_KEY_OR_TOKEN,
  // The US-only region group, for example
  domain: db.us.fauna.com
})
```

reference:
* connections page: https://docs.fauna.com/fauna/current/drivers/connections?lang=javascript
* region groups page: https://docs.fauna.com/fauna/current/api/fql/region_groups#how-to-use-region-groups

**Comment by ptpaterson** [comment]
My example was for javascript.  This template should be language agnostic.

**Comment by rts-rob** [comment]
Fixed in #2
